google-cloud-datastore/src/main/java/com/google/cloud/datastore/spi/v1/HttpDatastoreRpc.java

```diff
@@ -40,6 +39,7 @@
 import com.google.datastore.v1.RunAggregationQueryResponse;
 import com.google.datastore.v1.RunQueryRequest;
 import com.google.datastore.v1.RunQueryResponse;
+import io.opencensus.trace.Tracing;
```

Comment:
Do we need to add this?

Reply:
Due to removal of the datastore.TraceUtil class, this needs to be added to continue supporting opencensus tracing for HTTP.